src/main/kotlin/com/reicheltp/celtic_rituals/rituals/bowl/BowlRitualRecipe.kt

```diff
@@ -92,7 +94,10 @@ class BowlRitualRecipe(
 
             val duration = json.get("duration")?.asInt ?: DEFAULT_DURATION
 
-            return BowlRitualRecipe(recipeId, ingredients, result, duration)
+            return BowlRitualRecipe(
+                recipeId, ingredients, result, duration,
+                json.get("color")?.asColor ?: -1
```

Comment:
We should introduce a val color here like it is done with the duration parameter to stay consistent.